doc/classes/String.xml

```diff
@@ -551,6 +551,14 @@
 				[b]Note:[/b] This method checks identifiers the same way as GDScript. See [method TextServer.is_valid_identifier] for more advanced checks.
 			</description>
 		</method>
+		<method name="is_whitespace" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns true if the string is formed only of 0 or more whitespace characters.
+
+				The list of whitespace characters can be found [url=https://en.wikipedia.org/wiki/Whitespace_character]here[/url] (only Is_Whitespace is considered).
```

Comment:
I am not sure what `(only Is_Whitespace is considered)` means here. There's no mention of this in the linked article.
```suggestion
				Returns [code]true[/code] if this string is only composed of whitespace characters. For a full list of whitespace characters, see [url=https://en.wikipedia.org/wiki/Whitespace_character]here[/url].
```